genienlp/ned/scripts/analyze_bootleg_results.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+from collections import Counter
+
+import jsonlines
+from termcolor import colored
+
+from genienlp.ned.bootleg import BatchBootlegEntityDisambiguator
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument('--input_file', type=str)
+parser.add_argument('--database_dir', type=str)
+parser.add_argument('--ned_domains', type=str, nargs='+')
+parser.add_argument('--subsample', type=int, default='1000000000')
+
+
+args = parser.parse_args()
+
+if __name__ == '__main__':
```

Comment:
Why is this not a normal genienlp command?

Reply:
These scripts are for local data analysis. I don't want to crowd genienlp commands with data processing/ analysis scripts.

Reply:
This is a generally usable script that could be part of everyone's workflow. It's not a custom one-off script. I think it's worth making it part of genienlp proper.

Reply:
My main concern is this will be an invitation for more scripts. As a general rule, I think we should exclude data manipulation scripts that do not rely on a specific model/ tokenizer and have them in genie-toolkit or workdirs. I can add these for now since you and Sina are both in favor and Kevin may wanna take a look too.

Reply:
I agree, anything that's project/model specific should go into genie-workdirs.